app/controllers/declaration/CommodityMeasureController.scala

```diff
@@ -63,6 +63,15 @@ class CommodityMeasureController @Inject()(
       )
   }
 
+  private def form()(implicit request: JourneyRequest[_]) = CommodityMeasure.form(request.declarationType).withSubmissionErrors()
+
+  private def getCommodityCode(itemId: String)(implicit request: JourneyRequest[_]) =
+    for {
+      exportItem <- request.cacheModel.itemBy(itemId)
+      commodityDetails <- exportItem.commodityDetails
+      commodityCode <- commodityDetails.combinedNomenclatureCode
+    } yield commodityCode
```

Comment:
I'd suggest placing this method in `ExportsDeclaration`. There are more similar to this one already.

Reply:
:+1: